src/js/shariff.js

```diff
@@ -257,9 +257,12 @@ module.exports = Shariff;
 // export Shariff class to global (for non-Node users)
 global.Shariff = Shariff;
 
-// initialize .shariff elements
-$('.shariff').each(function() {
-    if (!this.hasOwnProperty('shariff')) {
-        this.shariff = new Shariff(this);
-    }
+$(global.document).ready(function () {
```

Comment:
As if jQuery code wouldn't be implicit enough yet ;)

While I will of course change it, making it a separate fix won't work as for the XHTML/XHTML5 support this change in execution is a prerequisite.